### PR TITLE
Clinic validation

### PR DESCRIPTION
**Pull request by gsvr**
Use fake clinic codes in testing and QA config file.  Real clinic codes to be put in private repo.

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

**Comment by imsickofmaps** [comment]
@smn ready for review - actual clinic codes will be in the praekelt/go-ndoh-private

File: src/clinic.js [hunk 1]
```diff
@@ -125,9 +125,20 @@ go.app = function() {
         });
 
         self.states.add('states_clinic_code', function(name) {
+            var error = $('Sorry, the clinic number did not validate. ' +
+                          'Please reenter the clinic number:');
```
Comment (smn) [inline src/clinic.js:129]: I'm always confused if this requires a dash or not.

**Comment by smn** [comment]
:+1:

File: src/clinic.js (hunk 1 above)
Comment (imsickofmaps) [inline src/clinic.js:129]: http://www.merriam-webster.com/dictionary/reenter

Comment (gsvr) [inline src/clinic.js:129]: It seems like both reenter and re-enter are okay if you look at online
dictionaries.  Personally I think we may want to think of changing the text
to something like:

"Sorry, {{clinic_number_previously_entered}} is not a valid option.
Please type the clinic number again."

That way they can hopefully see what was wrong with what they typed.  Just
my 2 cents.

Comment (smn) [inline src/clinic.js:129]: Nice comment on the bottom of that page: http://www.merriam-webster.com/dictionary/reenter

**Pull request merged**